xblock/reference/__init__.py

```diff
@@ -3,3 +3,4 @@
 
 See README.md for details.
 """
+from __future__ import unicode_literals
```

Comment:
On an empty `__init__`?  Wouldn't it be easier just to put a `u` in front of the docstring?